docs/changelog/101265.yaml

```diff
@@ -0,0 +1,13 @@
+pr: 101265
```

Comment:
Not 100% sure if I should include this here or in the 8.11 backport. I've reached out to es-delivery to check. Related convo (sorry — internal link only): https://elastic.slack.com/archives/C01EMR8S1KN/p1698157086067489

Reply:
This is good to leave as is.